pkg/apis/istio/v1alpha1/fixup_structs/main.go

```diff
@@ -120,21 +128,14 @@ func main() {
 	}
 }
 
-// patchValues is helper function to patch specific fields of the generated values_types.pb.go
+// patchValues is helper function to patch specific naming of fields of the generated values_types.pb.go
+// some fields namings are inconsistent between proto file and values.yaml file, eg. istio-ingressgateway
+// is not a valid field name in proto but used in values.yaml.
```

Comment:
You can move some of this text as the explanation for the map where I left the other comment.

Reply:
done